spec/datadog/core/telemetry/v1/app_event_spec.rb

```diff
@@ -98,6 +101,18 @@
       end
     end
 
+    context 'when :install_signature' do
+      context 'is nil' do
+        let(:install_signature) { nil }
+        it { is_expected.to be_a_kind_of(described_class) }
+      end
+
+      context 'is InstallSignature' do
+        let(:integrations) { Datadog::Core::Telemetry::V1::InstallSignature.new(install_id: '123', install_type: 'docker', install_time: '2020-01-01') }
```

Comment:
I would use UNIX timestamps instead of YYYY-MM-DD dates for `install_time` to avoid creating confusion

Reply:
Ah, yeah, I can do that. Much of the PR was auto-completed by copilot so I breezed past that bit. 😅